.github/workflows/release-candidate.yaml

```diff
@@ -17,9 +17,6 @@ jobs:
     runs-on: ubuntu-18.04
     if: github.repository_owner == 'Apicurio'
     steps:
-      - name: Checkout Code With Ref '${{ github.ref }}'
```

Comment:
There are multiple checkouts of different codebases, and the github action that we use to create github release works only the root directory. Hence, we were checkout out the registry code in root as well to create a github release. This might lead to inconsistency.